modules/core/src/main/java/org/apache/ignite/configuration/IgniteConfiguration.java

```diff
@@ -566,11 +574,22 @@
     private DataStorageConfiguration dsCfg;
 
     /** Active on start flag. */
+    @Deprecated
     private boolean activeOnStart = DFLT_ACTIVE_ON_START;
 
+    /** Indicates that activeOnStart property was set explicitly. */
+    private boolean activeOnStartPropSetFlag;
```

Comment:
redundant

Reply:
Nope) See `ActiveOnStartPropertyTest`, `ClusterStateOnStartPropertyTest` tests

Reply:
still see nothing.

Reply:
We must warn user if he uses together 'activeOnStart` and 'stateOnStart` properties. See `GridClusterStateProcessor#start`

Reply:
It's a QA team requirement. See https://ggsystems.atlassian.net/browse/GG-24476 test 1.7 the last scenario point. I don't have any idea about how to satisfy this requirement. I've discussed this solution with @glukos and he was ok with them.